### PR TITLE
Resovendo issue 4

### PR DESCRIPTION
**Pull request by deciolh**

### CODE REVIEW

**Comment by eminetto** [comment]
Hello, @deciolh! This is your first Pull Request that will be reviewed by [Ebert](https://ebertapp.io/), an automatic Code Review service. It will leave comments on this diff with potential issues and style violations found in the code as you push new commits. You can also see all the issues found on this Pull Request on its [review page](https://ebertapp.io/github/Coderockr/exercicio-git/pulls/24). Please check our [documentation](https://ebertapp.io/docs) for more information.

File: tests/JokenPoTest.php [hunk 1]
```diff
@@ -26,4 +32,3 @@ public function testRules()
         $this->assertContains($rules, $jokenpo->rules());
     }
 }
-
```
Comment (jeanCarloMachado) [inline tests/JokenPoTest.php]: Não  pode remover essa linha.

File: _reports/logs/junit.xml [hunk 2]
```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <testsuites>
-  <testsuite name="Test Suite" tests="3" assertions="3" errors="0" failures="0" skipped="0" time="0.004182">
```
Comment (jeanCarloMachado) [inline _reports/logs/junit.xml]: É melhor não mandar esse arquivo.

File: src/JokenPo/JokenPo.php [hunk 3]
```diff
@@ -9,6 +9,9 @@ public function run(string $play1, string $play2)
         if ($play1 == $play2) {
             return 'empate';
         }
+        elseif ($play1 == 'papel' and $play2 == 'pedra') {
```
Comment (jeanCarloMachado) [inline src/JokenPo/JokenPo.php]: Melhor deixar o elseif na mesma linha do fecha chaves acima.

**Comment by deciolh** [comment]
@jeanCarloMachado podes verificar?
Obrigado.

**Pull request merged**